internal/policy/policy_test.go

```
@@ -747,39 +798,39 @@ func TestEffectiveTimeAttestationAllowMutation(t *testing.T) {
 func TestAttestationTime(t *testing.T) {
 	cases := []struct {
 		name                      string
-		choosenTime               string
+		chosenTime               string
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(gci)